apps/app/src/views/routes/Settings/components/templates/TemplateForm/components/CtaButtons.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, HStack } from '@chakra-ui/react';
+
+export const CtaButtons = ({
+  edit,
+  handleCancel,
+  loading,
+  isSubmitting,
+  setSubmitting,
+  catalogs
+}: any) => (
+  <HStack alignSelf="end">
+    <Button variant="outline" onClick={handleCancel} isLoading={loading && isSubmitting}>
+      Cancel
+    </Button>
+    <Button
+      type="submit"
+      colorScheme="brand"
+      form="template-builder"
+      onClick={() => setSubmitting(true)}
+      disabled={loading || catalogs.loading}
```

Comment:
not urgent, but jomi wants to move away from disabled states and towards it triggering toast messages that inform you what's missing

Reply:
We'll deal with this later